src/styles/brackets_codemirror_override.less

```diff
@@ -27,6 +27,7 @@
 
     > * {
         text-indent: 0px;
+        margin-left: -4px;
```

Comment:
@petetnt could this be done as part of [the hack itself](https://github.com/adobe/brackets/pull/11909/files#diff-3b7e955135e4a77b20767ed40e551395R990) as opposed to stylesheet? this way the value of the shift (`4px`) will be shared and not duplicated.